jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TravelerSerializer.java

```diff
@@ -146,25 +147,24 @@ else if (headingSlice.equals("allHeadings")) {//All bits were set
         {
             msgId.setChosenFlag(MsgId.furtherInfoID_chosen);
 
-            //Not Sure where this @ToDO
             msgId.setFurtherInfoID(new FurtherInfoID(new byte[] { 0x00,0x00 }));
 
             p1.add(travelerInfoType);
             dataFrame.setFrameType(TravelerInfoType.valueOf(Long.parseLong(travelerInfoType)));
 
-            String minuteOfTheYear = ob.getJSONObject("header").getString("MinuteOfTheYear");
+            String minuteOfTheYear = ob.getJSONObject(header).getString("MinuteOfTheYear");
```

Comment:
Repeating a prior comment, let's check with Tony E of WYDOT team and if he agrees, we should use an ISO time string for startTime (and call it startTime instead of MinutesOfTheYear). We should the use TimeUtils methods to convert that timestamp to a zoned time. 

For MinutesDuration, we should also use StopTime in my opinion to remove the burden of calculating the duration in minutes from the client application. But again, let's confirm with WYDOT team if they prefer one or the other. If we keep MinutesDuration, we shoudl rename it to start with lowercase as in minutesDuration or simply duration. By the same token, SignPriority should be "priority" or "signPriority" for consistency sake.